homeassistant/components/media_player/webostv.py

```diff
@@ -100,7 +112,8 @@ def request_configuration(host, hass, add_devices):
     # pylint: disable=unused-argument
     def lgtv_configuration_callback(data):
         """The actions to do when our configuration callback is called."""
-        setup_tv(host, hass, add_devices)
+        # pylint: disable=no-value-for-parameter
```

Comment:
Please remove this directive. If it is needed it means you have a bug, if it is not needed it will prevent PyLint from finding bugs.